src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +373,227 @@ public void SetFocus(bool focused)
+            {
+                return Interop.ChromiumEwk.ewk_view_scale_get(_realHandle);
+            }
+        }
+
+        /// <summary>
+        /// Sets the current scale factor of the page.
+        /// </summary>
+        /// <param name="scaleFactor,">A new level to set.</param>
+        /// <param name="p">The class Point object with X, Y coordinates.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewScale (double scaleFactor, Point p)
+        {
+            return Interop.ChromiumEwk.ewk_view_scale_set(_realHandle, scaleFactor, p.X, p.Y);
+        }
+
+        /// <summary>
+        /// Sets the current page's visibility.
+        /// </summary>
+        /// <param name="enable">'true' to set on the visibility of the page, 'false' otherwise.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewVisibility (bool enable)
+        {
+            return Interop.ChromiumEwk.ewk_view_visibility_set(_realHandle, enable);
+        }
+
+        /// <summary>
+        /// Get and Sets the scroll position of the page.
+        /// </summary>
+        /// <returns>The class Point object with X, Y coordinates.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Point ScrollPos
+        {
+            get
+            {
+                Point p;
+                Interop.ChromiumEwk.ewk_view_scroll_pos_get(_realHandle, out p.X, out p.Y);
+                return p;
+            }
+            set
+            {
+                Interop.ChromiumEwk.ewk_view_scroll_set(_realHandle, value.X, value.Y);
+            }
+        }
+
+        /// <summary>
+        /// Scrolls the webpage of view by x and y.
+        /// </summary>
+        /// <param name="p">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ScrollBy (Point p)
+        {
+            Interop.ChromiumEwk.ewk_view_scroll_by(_realHandle, p.X, p.Y);
+        }
+
+        /// <summary>
+        /// Searches and highlights the given text string in the document.
+        /// </summary>
+        /// <param name="text">The text to find.</param>
+        /// <param name="options">The options to find.</param>
+        /// <param name="maxMatchCount">The maximum match count to find, unlimited if 0.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool FindText (string text, FindOptions options, int maxMatchCount)
+        {
+            return Interop.ChromiumEwk.ewk_view_text_find(_realHandle, text, (Interop.ChromiumEwk.FindOptions)options, maxMatchCount);
+        }
+
+        /// <summary>
+        /// Enumeration for Http Method.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public enum Method
```

Comment:
How about HttpMethod?
And plz move this out of WebView class.

Reply:
Done